pydo/database/sqlite.py

```diff
@@ -0,0 +1,53 @@
+from .database import Database
+import sqlite3
+from os import path
+from ..config.config import dump_config
+
+
+class SqliteDatabase(Database):
+    db_ext = '.db'
+
+    def __init__(self, name, db_path, config_path=None):
+        super().__init__(name, db_path, config_path)
+        self.cursor = None
+        self.connection = None
+
+        if self.name.endswith(SqliteDatabase.db_ext):
+            self.complete_path = path.join(self.path, self.name)
+        else:
+            self.complete_path = path.join(self.path,
+                                           self.name + SqliteDatabase.db_ext)
+
+    def initialise(self):
+        self.load()  # Connecting to non-existent db file creates a new db file
+        self.cursor.execute("""CREATE TABLE tasks (
```

Comment:
For when you have more big static SQL strings, it'd be worth putting them all into a separate module